extensions/linz/README.md

```diff
@@ -81,25 +81,25 @@ See [ISO/IEC 13249-3:2016(en)](https://www.iso.org/obp/ui/#!iso:std:60343:en) fo
-| name       | string    | For the `manager` role this should be either the Toitū Te Whenua LINZ role name of a person who manages this dataset (to preserve their privacy) or the name of a team that manages this dataset. |
-| roles      | \[string] | **There needs to be at least one provider with the `custodian` role and one with the `manager` role.**                                                                                            |
-| url        | string    | The `url` should be an internal URL that links to more information about that person or team.                                                                                                     |
+| Property Name | Type      | Description                                                                                                                                                                                       |
+| ------------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| name          | string    | For the `manager` role this should be either the Toitū Te Whenua LINZ role name of a person who manages this dataset (to preserve their privacy) or the name of a team that manages this dataset. |
+| roles         | \[string] | **There needs to be at least one provider with the `custodian` role and one with the `manager` role.**                                                                                            |
+| url           | string    | The `url` should be an internal URL that links to more information about that person or team.                                                                                                     |
 
 ### Provider Object
 
 This expands on the [provider object in the STAC spec](https://github.com/radiantearth/stac-spec/blob/v1.0.0/item-spec/common-metadata.md#provider-object). Only differences from that definition are mentioned here.
 
-| Field Name | Type      | Description                                                                                            |
-| ---------- | --------- | ------------------------------------------------------------------------------------------------------ |
-| roles      | \[string] | **There needs to be at least one provider with the `licensor` role and one with the `producer` role.** |
+| Property Name | Type      | Description                                                                                            |
+| ------------- | --------- | ------------------------------------------------------------------------------------------------------ |
+| roles         | \[string] | **There needs to be at least one provider with the `licensor` role and one with the `producer` role.** |
 
-## Asset Fields
+## Asset Properties
```

Comment:
I don't think assets have properties. Fields is fine here I think, https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md#asset-object